modules/auxiliary/scanner/http/epmp1000_dump_hashes.rb

```diff
@@ -104,10 +104,7 @@ def hash_dump(config_uri, cookie)
       )
 
       if good_response
-        vprint_status('++++++++++++++++++++++++++++++++++++++')
         vprint_status("#{rhost}:#{rport} - Dumping password hashes")
-        print_line("#{res.body}")
```

Comment:
I think this would stay.

Reply:
So, things like `print_line` tend to look all weird when you have a scan run with a lot of targets -- they're undecorated, don't mention the host, and can get all jumbled up with different response times and threading. I'd rather keep them out. 

They're fine for debugging and seeing what exactly is happening, but for production, I don't think they're helpful.

Reply:
I agree. It does that for multiple hosts. Removing it will keep the module run neat. +1